src/_roffio/tokenizer/text_roff_body_tokenizer.py

```diff
@@ -0,0 +1,166 @@
+from _roffio.tokenizer.abstract_roff_body_tokenizer import AbstractRoffBodyTokenizer
+from _roffio.tokenizer.combinators import bind, one_of, repeated
+from _roffio.tokenizer.common import tokenize_word
+from _roffio.tokenizer.errors import TokenizationError
+from _roffio.tokenizer.token import Token
+from _roffio.tokenizer.token_kind import TokenKind
+
+
+class TextRoffBodyTokenizer(AbstractRoffBodyTokenizer):
+    def __init__(self, stream):
+        """
+        :param stream: A byte stream containing roff data.
+        """
+        self._stream = stream
+
+    @property
+    def tokenize_delimiter(self):
+        return repeated(one_of(self.tokenize_comment, self.tokenize_space))
+
+    @property
+    def stream(self):
+        return self._stream
+
+    @property
+    def tokenize_keyword(self):
+        return {
+            kw_kind: bind(
+                self.tokenize_delimiter,
+                tokenize_word(self.stream, kw, kw_kind),
+            )
+            for kw_kind, kw in TokenKind.keywords().items()
+        }
+
+    @property
+    def tokenize_array_size(self):
+        return self.tokenize_numeric_value
+
+    @property
+    def tokenize_value(self):
+        return one_of(self.tokenize_numeric_value, self.tokenize_string_literal)
+
+    def swap_endianess(self):
+        pass
+
+    def tokenize_numeric_value(self):
+        """
+        Tokenize any text numeric value, yields
+        Token(TokenKind.NUMERIC_VALUE, 0, 3) for stream
+        containing "1.0".
+        """
+        yield from self.tokenize_delimiter()
+        start = self.stream.tell()
+        end = start
+        read_char = self.stream.read(1)
+        if read_char and read_char.isnumeric() or read_char == "-":
+            while read_char and (read_char.isnumeric() or read_char in ".eE+-"):
+                end = self.stream.tell()
+                read_char = self.stream.read(1)
+        if end - start < 1:
```

Comment:
I don't think  we can have `end - start < 0` here so can probably simplify for readability:
```suggestion
        if end == start:
```